.github/workflows/tests.yml

```diff
@@ -0,0 +1,32 @@
+name: Test 
+
+on: 
+  pull_request: 
+    branches: 
+      - base 
```

Comment:
This will have to be development. We want tests to run when we make a PR against dev.

Reply:
I'm pretty sure that's why the builds kept failing. Once I corrected line 6, everything passed!